spylon_kernel/scala_interpreter.py

```diff
@@ -391,8 +385,11 @@ def _interpret_sync(self, code, synthetic=False):
         ScalaException
             When there is a problem interpreting the code
         """
+        # Ensure the cell is not incomplete. Same approach taken by Apache Zeppelin.
+        code = 'print("")\n'+code
+
         try:
-            res = self.jimain.interpret(code, synthetic)
+            res = self.jimain.interpret(code, False)
```

Comment:
Nothing in the codebase here ever set it to True that I could find, so I preferred to remove it as YAGNI. As for what the parameter means, I haven't found anything in the API doc (http://www.scala-lang.org/api/2.12.1/scala-compiler/scala/tools/nsc/interpreter/IMain.html) but http://docs.scala-lang.org/glossary/#synthetic-class might apply and explain why False is the correct value. That's a guess, at best.